contributor-guide/modules/ROOT/nav.adoc

```diff
@@ -1,16 +1,19 @@
 * New Content
+* xref:index.adoc[Contributor Guide]
 * xref:intro.adoc[Introduction to becoming a Boost Contributor]
+* xref:antora.adoc[Antora Guide]
 * Legacy Content
-** xref:documentation-structure.adoc[Writing Documentation for Boost - Documentation Structure]
-** xref:index-file-template.adoc[Index File Template]
+** xref:documentation-overview.adoc[Writing Documentation for Boost - Documentation Structure]
+** xref:documentation-structure-guidelines.adoc[Boost Library Documentation Guidelines]
 ** Templates
-*** xref:templates/acknowledgements-template.adoc[Acknowledgments Template]
-*** xref:templates/bibliography-template.adoc[Bibliography Template]
-*** xref:templates/configuration-template.adoc[Configuration Template]
-*** xref:templates/definitions-template.adoc[Definitions Template]
```

Comment:
You don't need to repeat the section name here unless the title is different. You can define the names you want directly in the page:

```adoc
= Section Name
:reftext: Different Section Name
:navtitle: Different Section Name used in the navbar
```